docs/configuration.md

```diff
@@ -49,67 +49,68 @@ Unless you need to set a non-default value, it is recommended to only populate o
+| `ETH_GET_BLOCK_BY_RESULTS_BATCH_SIZE`       | "25"                                  | The number of contract results to request from the Mirror Node per batch durin an eth_getBlockByHash or eth_getBlockByNumber call                                                                                                                                                                      |
+| `ETH_GET_GAS_PRICE_CACHE_TTL_MS`            | "1_800_000"                           | Time in ms to cache ethGasPrice returned                                                                                                                                                                                                                                                               |
+| `ETH_CALL_DEFAULT_TO_CONSENSUS_NODE`        | "false"                               | Flag to set if eth_call logic should first query the mirror node.                                                                                                                                                                                                                                      |
+| `ETH_CALL_CONSENSUS_SELECTORS`              | [""]                                  | A comma-separated list of special transaction selectors that should always be routed to the Consensus node.                                                                                                                                                                                            |
+| `ETH_GET_LOGS_BLOCK_RANGE_LIMIT`            | "1000"                                | The maximum block number range to consider during an eth_getLogs call.                                                                                                                                                                                                                                 |
+| `ETH_GET_TRANSACTION_COUNT_MAX_BLOCK_RANGE` | "1000"                                | The maximum number of transactions to return when running eth_getBlockByHash or eth_getBlockByNumber with transaction objects set to true call.                                                                                                                                                        |
+| `HEDERA_SPECIFIC_REVERT_STATUSES`           | ["WRONG_NONCE", "INVALID_ACCOUNT_ID"] | The maximum number of transactions to return when running eth_getBlockByHash or eth_getBlockByNumber with transaction objects set to true call.                                                                                                                                                        |
```

Comment:
Description doesn't match the env type or logic

Reply:
Fixed.